packages/features/bookings/lib/getBookingInfo.ts

```diff
@@ -9,9 +9,14 @@ const getBookingInfo = async (uid: string) => {
     return { bookingInfoRaw: undefined, bookingInfo: undefined };
   }
 
+  let previousBooking = null;
+  if (bookingInfoRaw.fromReschedule) {
+    previousBooking = await getUserBooking(bookingInfoRaw.fromReschedule);
```

Comment:
We shouldn't do this:
- `getUserBooking` is heavy, it pulls in lots of data
- I see that all we need for UI are `uid` and `rescheduledBy`